### PR TITLE
UUID replaces Timestamp in the notebook template

### PR DESCRIPTION
**Pull request by krishr2d2**
Suggesting changes to the notebooks/notebook_template.ipynb file.

Replaces TIMESTAMP with the UUID to avoid naming conflicts during regression-tests.

### CODE REVIEW

**Comment by soheilazangeneh** [comment]
I like the idea of UUID instead of Timestamp. I am just wondering what kind of naming conflict will happen in regression testing. Can you give me an example?

**Comment by krishr2d2** [comment]
> I like the idea of UUID instead of Timestamp. I am just wondering what kind of naming conflict will happen in regression testing. Can you give me an example please?

Since the tests happen parallelly for a bunch of notebooks at a time, there was a scenario where the timestamp(suffix) generated while creating a cloud storage bucket matched in two of the notebooks. Eventually, one of them threw a "bucket not found" exception as the same bucket was cleared by the other notebook by then.

**Comment by soheilazangeneh** [comment]
> 

Got it! Thanks!

**Review by soheilazangeneh — APPROVED** [review]
LGTM!

**Review by ivanmkc — CHANGES_REQUESTED** [review]
Logs show a failure:

```
InvalidUrlError: Invalid bucket name in URL "python-docs-samples-testsaip-f5ef7c0f-2095-4725-9e60-dc3bd9556353".
```

**Comment by soheilazangeneh** [comment]
> 

Where can I find the log containing this error?

File: notebooks/notebook_template.ipynb [hunk 1]
```diff
@@ -373,9 +373,11 @@
         "id": "06571eb4063b"
```
Comment (ivanmkc) [inline notebooks/notebook_template.ipynb:373]: <p>Use </p><p> </p><pre class="ql-syntax" spellcheck="false">def generate_uuid(length: int = 8) -&gt; str: 
  return ''.join(random.choices(string.ascii_lowercase + string.digits, k=length))
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/745/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Comment (krishr2d2) [reply notebooks/notebook_template.ipynb:373]: Added!!

**Review by ivanmkc — APPROVED** [review]

**Pull request merged**